simcore/simulation/library/crosslink.cpp

```diff
@@ -1,4 +1,5 @@
 #include "crosslink.hpp"
+#include <iostream>
```

Comment:
If this was added for debugging purposes, it should be removed after. Also, you should stick to using the Logger static methods as good practice.